bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/Clip2Bridge.java

```diff
@@ -399,7 +398,7 @@ public void onGoAway(@Nullable Session session, @Nullable GoAwayFrame frame) {
             Objects.requireNonNull(session);
             if (session.equals(http2Session)) {
                 Thread recreateThread = new Thread(() -> recreateSession(),
-                        "OH-binding-" + HueBindingConstants.BINDING_ID + "-Clip2Bridge");
+                        "OH-binding-" + bridgeHandler.getThing().getUID() + "-Clip2Bridge");
```

Comment:
Perhaps `"OH-binding-" + bridgeHandler.getThing().getUID() + "-RecreateSession")` is better. But otherwise looks good.

Reply:
Changed.